peer/brontide.go

```diff
@@ -733,7 +733,15 @@ func NewBrontide(cfg Config) *Brontide {
 		IntervalDuration: p.scaleTimeout(pingInterval),
 		TimeoutDuration:  p.scaleTimeout(pingTimeout),
 		SendPing: func(ping *lnwire.Ping) {
-			p.queueMsg(ping, nil)
+			// We'll provide an errChan so we can make this a fully
```

Comment:
hmm but the ping timeout is still set before, so not sure what this blocking call changes tho:

```
// Set up our bookkeeping for the new Ping.
			if err := m.setPingState(pongSize); err != nil {
				m.cfg.OnPongFailure(err)

				return
			}

			m.cfg.SendPing(ping)
```
?

Reply:
Ah yeah need to re-order that as well!